src/RadixDlt.NetworkGateway.PostgresIntegration/Models/EntityResourceAggregateHistory.cs

```diff
@@ -84,12 +86,41 @@ internal class EntityResourceAggregateHistory
     [Column("fungible_resource_entity_ids")]
     public List<long> FungibleResourceEntityIds { get; set; }
 
-    [Column("fungible_resource_last_update_state_versions")]
-    public List<long> FungibleResourceLastUpdateStateVersions { get; set; }
+    [Column("fungible_resource_significant_update_state_versions")]
```

Comment:
I could come with better name for these columns so I think we have to leave them as it is. But if we can't find better name, what do you think about adding comment here to explaining what it is about?

Reply:
Refactored EntityResourceAggregateHistory quite heavily.